README.md

```diff
@@ -41,8 +41,11 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
   "name": "Eddie Jaoude",
   "username": "eddiejaoude"
 }
+ 
```

Comment:
Looks like the prettier is complaining about this line. What should we do about it?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Tough question can we do how ignore empty lines of n prettier?

Reply:
What about this only (without changing the above code example)

---
> Also add a new blank line at the end of the code after `}`.

Reply:
or

---

> Also add a new line at the end of the code after `}` and leave it empty.

Reply:
last one 

---
> Also add a new line at the end of the code after `}` and leave it empty. Like: 👇
```
1. {
2.   "image": "https://user-images.githubusercontent.com/624760/91057573-48531300-e61f-11ea-9e13-2d7384e42000.png",
3.   "issueId": 66,
4.   "name": "Eddie Jaoude",
5.   "username": "eddiejaoude"
6. }
7.
```

Reply:
#### what about these 3 examples (without changing the above JSON code example)

---
1. 
> Also add a new blank line at the end of the code after `}`.
---
2. 
> Also add a new line at the end of the code after `}` and leave it empty.
---
3. 
> Also add a new line at the end of the code after `}` and leave it empty. Like: 👇
```
1. {
2.   "image": "https://user-images.githubusercontent.com/624760/91057573-48531300-e61f-11ea-9e13-2d7384e42000.png",
3.   "issueId": 66,
4.   "name": "Eddie Jaoude",
5.   "username": "eddiejaoude"
6. }
7.
```